test/verify/check-pages

```diff
@@ -178,11 +178,11 @@ OnCalendar=daily
         self.open_lang_modal()
         b.click('#display-language-modal li[data-value=de-de] button')
         b.click("#display-language-modal footer button.pf-m-primary")
-        b.expect_load()
         if b.cdp.browser.name == "chromium":
             # HACK: work around language switching in Chrome not working in current session (issue #8160)
             b.reload(ignore_cache=True)
         b.wait_visible("#content")
+        b.wait_visible("#host-apps li a:contains('Dienste')")
```

Comment:
This does not belong into the "Allow exceptions while waiting" commit.

Reply:
Removed.